programmers/정렬.K번째수/just-stopyoon.py

```diff
@@ -0,0 +1,11 @@
+def solution(array, commands):
+    answer = []
+    for a in range(len(commands)):
+        i = commands[a][0] - 1 # i번째 숫자
+        j = commands[a][1] # j번째 숫자 (슬라이싱 고려해서 -1 안함)
+        k = commands[a][2] - 1 # 배열은 0부터 시작하므로 1 빼기
```

Comment:
for문을 사용할 때 a를 commands 배열의 index로만 사용하고 있으니까, range()를 사용하지 않고 바로 배열에서 원소를 가져오는 것도 하나의 방법일 것 같아요!

```suggestion
    for command in commands:
        i = command[0] - 1 # i번째 숫자
        j = command[1] # j번째 숫자 (슬라이싱 고려해서 -1 안함)
        k = command[2] - 1 # 배열은 0부터 시작하므로 1 빼기
```

변수명을 이해할 수 있게 작성해서 가독성이 좋은 것 같습니다!
수고하셨습니다 ~~!